instantshare.spec

```diff
@@ -0,0 +1,51 @@
+# -*- mode: python -*-
+
+block_cipher = None
+
+hidden_imports = ['cli.audio', 
+                  'cli.file', 
+                  'cli.gui', 
+                  'cli.screen', 
+                  'cli.text', 
+                  'cli.video',
+                  'screenshot.snippingtool',
+                  'screenshot.windows_tk',
+                  'screenshot.gnome_screenshot',
+                  'storage.dropbox',
+                  'storage.googledrive',
+                  'storage.imgur',
+                  'storage.owncloud',
+                  'storage.pastebin',
+                  'storage.sftp',
+                  'storage.twitter_intent']
+
+added_files = [('src/res', 'res')]
+
+a = Analysis(['src\\instantshare'],
+             pathex=['C:\\Developer\\instantshare'],
```

Comment:
These lines lead to problems on linux based OSs. 
```
pyinstaller instantshare.spec 
35 INFO: PyInstaller: 3.2.1
35 INFO: Python: 3.6.2
36 INFO: Platform: Linux-4.12.4-1-ARCH-x86_64-with-arch
37 INFO: UPX is not available.
Traceback (most recent call last):
  File "/usr/bin/pyinstaller", line 11, in <module>
    load_entry_point('PyInstaller==3.2.1', 'console_scripts', 'pyinstaller')()
  File "/usr/lib/python3.6/site-packages/PyInstaller/__main__.py", line 90, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "/usr/lib/python3.6/site-packages/PyInstaller/__main__.py", line 46, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 788, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 734, in build
    exec(text, spec_namespace)
  File "<string>", line 34, in <module>
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 162, in __init__
    raise ValueError("script '%s' not found" % script)
ValueError: script 'src\instantshare' not found
```

Changing `src\\instantshare` to `src/instantshare` fixes that, but now I get:

```
pyinstaller instantshare.spec
34 INFO: PyInstaller: 3.2.1
35 INFO: Python: 3.6.2
35 INFO: Platform: Linux-4.12.4-1-ARCH-x86_64-with-arch
37 INFO: UPX is not available.
38 INFO: Extending PYTHONPATH with paths
['/home/dt/workspaces/python/instantshare/src',
 '/home/dt/workspaces/python/instantshare/C:\\Developer\\instantshare']
38 INFO: checking Analysis
38 INFO: Building Analysis because out00-Analysis.toc is non existent
38 INFO: Initializing module dependency graph...
41 INFO: Initializing module graph hooks...
42 INFO: Analyzing base_library.zip ...
Traceback (most recent call last):
  File "/usr/bin/pyinstaller", line 11, in <module>
    load_entry_point('PyInstaller==3.2.1', 'console_scripts', 'pyinstaller')()
  File "/usr/lib/python3.6/site-packages/PyInstaller/__main__.py", line 90, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "/usr/lib/python3.6/site-packages/PyInstaller/__main__.py", line 46, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 788, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 734, in build
    exec(text, spec_namespace)
  File "<string>", line 34, in <module>
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 212, in __init__
    self.__postinit__()
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/datastruct.py", line 161, in __postinit__
    self.assemble()
  File "/usr/lib/python3.6/site-packages/PyInstaller/building/build_main.py", line 317, in assemble
    excludes=self.excludes, user_hook_dirs=self.hookspath)
  File "/usr/lib/python3.6/site-packages/PyInstaller/depend/analysis.py", line 560, in initialize_modgraph
    graph.import_hook(m)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 1509, in import_hook
    source_package, target_module_partname, level)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 1661, in _find_head_package
    target_module_headname, target_package_name, source_package)
  File "/usr/lib/python3.6/site-packages/PyInstaller/depend/analysis.py", line 209, in _safe_import_module
    module_basename, module_name, parent_package)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 2077, in _safe_import_module
    module_name, file_handle, pathname, metadata)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 2167, in _load_module
    self._scan_code(m, co, co_ast)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 2585, in _scan_code
    module, module_code_object, is_scanning_imports=False)
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 2831, in _scan_bytecode
    global_attr_name = get_operation_arg_name()
  File "/usr/lib/python3.6/site-packages/PyInstaller/lib/modulegraph/modulegraph.py", line 2731, in get_operation_arg_name
    return module_code_object.co_names[co_names_index]
IndexError: tuple index out of range
```
I'm not sure whether or not that error is related to this, but look at the path extensions at the top. The second one does not seem to make any sense to me.